spec/lib/traject/config/access_spec.rb

```diff
@@ -26,163 +26,186 @@
   let(:client) { instance_double(FolioClient, statistical_codes: [], instance: {}) }
   let(:marc_record) { MARC::Record.new }
 
-  subject(:field) { result['access_facet'] }
-
-  context 'with an electronic resource' do
-    let(:marc_record) do
-      MARC::Record.new.tap do |r|
-        r.leader = '00988nas a2200193z  4500'
-        r.append(MARC::ControlField.new('008', '071214uuuuuuuuuxx uu |ss    u|    |||| d'))
-        r.append(MARC::DataField.new('956', '4', '0',
-                                     MARC::Subfield.new('u', 'http://caslon.stanford.edu:3210/sfxlcl3?url_ver=Z39.88-2004&amp;ctx_ver=Z39.88-2004&amp;ctx_enc=info:ofi/enc:UTF-8&amp;rfr_id=info:sid/sfxit.com:opac_856&amp;url_ctx_fmt=info:ofi/fmt:kev:mtx:ctx&amp;sfx.ignore_date_threshold=1&amp;rft.object_id=110978984448763&amp;svc_val_fmt=info:ofi/fmt:kev:mtx:sch_svc&amp;')))
+  describe 'access_facet' do
```

Comment:
This is mostly indenting changes.  Review may be easier if you select "Hide whitespace"